operator/charts/templates/clustertopology-validating-webhook-config.yaml

```
@@ -0,0 +1,34 @@
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: clustertopology-validating-webhook
+  labels:
+{{- include "operator.clustertopology.validating.webhook.labels" . | nindent 4 }}
```

Shouldn't this webhook configuration also be namespaced, like the other webhooks?

```suggestion
metadata:
  name: clustertopology-validating-webhook
  namespace: {{ .Release.Namespace }}
  labels:
{{- include "operator.clustertopology.validating.webhook.labels" . | nindent 4 }}
```